geomet/tests/esri_test.py

```diff
@@ -0,0 +1,205 @@
+    def test_loads_to_geojson_point(self):
+        """Tests Loading Esri Point Geometry to Point GeoJSON"""
+        self.assertEqual(esri.loads(esri_json_pt),
+                          {'type': 'Point', 'coordinates': (25282, 43770)})
+    def test_loads_to_geojson_multipoint(self):
+        """Tests Loading Esri MultiPoint Geometry to MultiPoint GeoJSON"""
+        self.assertEqual(esri.loads(esri_json_mpt),
+                          {'type': 'Multipoint', 'coordinates': [[-97.06138, 32.837], [-97.06133, 32.836], 
+                          [-97.06124, 32.834], [-97.06127, 32.832]]})
+    def test_loads_to_geojson_linstring(self):
+        """Tests Loading Esri Point Geometry to MultiLineString GeoJSON"""
+        self.assertEqual(esri.loads(esri_json_polylines),
+                          {'type': 'MultiLineString', 'coordinates': [[(-97.06138, 32.837), (-97.06133, 32.836), (-97.06124, 32.834), (-97.06127, 32.832)], 
+                          [(-97.06326, 32.759), (-97.06298, 32.755)]]})
+    def test_loads_to_geojson_polygon(self):
+        """Tests Loading Esri Polygon Geometry to MultiPolygon GeoJSON"""
+        self.assertEqual(esri.loads(esri_json_polygon),
+                         {'type': 'MultiPolygon', 'coordinates': [
+                             [[(-97.06138, 32.837), (-97.06133, 32.836), (-97.06124, 32.834), (-97.06127, 32.832), (-97.06138, 32.837)]], 
+                         [[(-97.06326, 32.759), (-97.06298, 32.755), (-97.06153, 32.749), (-97.06326, 32.759)]]
+                         ]})
+    def test_loads_empty_pt_to_gj(self):
+        """Tests Loading an empty Esri JSON Point to GeoJSON"""
+        geom = {"x":None,"y":None,"spatialReference":{"wkid":3857}}
+        self.assertEqual(esri.loads(geom),
+                          {'type': 'Point', 'coordinates': ()})
+    def test_loads_to_empty_mpt_to_gj(self):
+        """Tests Loading an empty Esri JSON MultiPoint to GeoJSON"""
+        geom = {
+            "points" : [],
+            "spatialReference" : {"wkid" : 4326}
+        }
+        geom_match = {'type': 'Multipoint', 'coordinates': []}
+        self.assertEqual(esri.loads(geom),
+                          geom_match)
+    def test_loads_empty_polyline_to_gj(self):
+        """Tests Loading an empty Esri JSON Polyline to GeoJSON"""
+        geom = {
+                "paths" : [],
+                "spatialReference" : {"wkid" : 4326}
+        }
+        geom_match = {'type': 'MultiLineString', 'coordinates': []}
+        self.assertEqual(esri.loads(geom),
+                          geom_match)
+    def test_loads_empty_polygon_to_gj(self):
+        """Tests Loading an empty Esri JSON Polygon to GeoJSON"""
+        geom = {
+            "rings" : [],
+            "spatialReference" : {"wkid" : 4326}
+        }
+        geom_match = {'type': 'MultiPolygon', 'coordinates': []}
+        self.assertEqual(esri.loads(geom),
+                          geom_match)
+
+class TestGeoJSONtoEsriJSON(unittest.TestCase):
```

Comment:
Can you add some tests proving correctness of the new `srid` parameter in the `dumps` function?

Reply:
@tomplex on the default to 4326, GeoJSON is designed for that coordinate system specifically.  So having it default to 4326 is good form I think.